keylime/cloud_verifier_common.py

```diff
@@ -374,8 +374,10 @@ def validate_agent_data(agent_data):
     if agent_data is None:
         return False, None
 
+    # validate that the ima_whitelist is proper JSON
+    lists = json.loads(agent_data['ima_whitelist'])
```

Comment:
I remember hitting some issues with json.loads, it was the way that sqlaclhemy was formatting the data which is why I used eval. I can't remember what it was now (which is not much use). I will try to remember what it was.

Reply:
Yeah, I didn't hit any issues, but `ast.literal_eval` has this warning: `Warning It is possible to crash the Python interpreter with a sufficiently large/complex string due to stack depth limitations in Python’s AST compiler.` which I was hitting with the large files.

Reply:
I do feel a bit 'antsy' about using eval, so if we can drop it lets do so.